src/app/components/faq/faq.component.html

```diff
@@ -9,7 +9,7 @@ <h1 class="my-5">Frequently Asked Questions</h1>
                 </h2>
             </div>
             <div [attr.id]="'faq_answer_'+i" class="collapse multi-collapse" [attr.aria-labelledby]="'heading'+i">
-                <div class="card-body" [innerHTML]="faq.answer"></div>
+                <div class="card-body" [innerHTML]="sanitizer.bypassSecurityTrustHtml(faq.answer)"></div>
```

Comment:
Have the sanitization in the component. It's better to keep all the logic to the ts files.

Reply:
Can you clarify? A lot of the usage of the `innerHTML` happens in an `*ngFor` in the template. I'm not sure if there is an ideal way to keep it in the ts files while we are using `*ngFor`.

Reply:
1. Add the attribute `safeAnswer: SafeHtml` to the `Faq` model.
2. Populate the `safeAnswer` using `faq.safeAnswer = sanitizer.bypassSecurityTrustHtml(faq.answer)` in the constructor.
3. Use `faq.safeAnwer` in the html
4. Refactor the model name `Faq` to `FAQ` because its an abbriviation.

Reply:
For other files, it might be easier to just add an attribute to the component.